grpc/swift-cubic/Package.resolved

```diff
@@ -1,70 +0,0 @@
-{
```

Comment:
I thought this file made it explicit which version of dependencies the project should use. What is your reasoning for not keeping it?